popup.html

```diff
@@ -1,72 +1,226 @@
-<!doctype html>
+<!DOCTYPE html>
 <html>
   <head>
     <title>Eztrackr</title>
-    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css">
+    <link
+      rel="stylesheet"
+      href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
+    />
     <script src="https://requirejs.org/docs/release/2.3.6/minified/require.js"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/fetch/2.0.3/fetch.js"></script>
     <style>
-        body{
-            width: 300px;
-            zoom: 0.8;
+      body {
+        width: 300px;
+        zoom: 0.8;
+      }
+      /* Style for the loader */
```

Comment:
- Can you create a separate CSS file for tall this styling? You can link it here instead.
 - I like how you did the CSS, but it would be better to use more Bootstrap I think? 
 - Decrease the padding of all the divs. In fact, just one small div with plain text also works. In the future, I might want to include add more stats so having a scrolling bar for 5 items already takes a lot of space.
 - Can you remove your bars loading animation and use bootstrap instead? I think it would look better. (https://getbootstrap.com/docs/4.4/components/spinners/) This would also decrease CSS content.